gemrb/GUIScripts/GUISTORE.py

```diff
@@ -1738,6 +1738,11 @@ def SetupItems (pc, Slot, Button, Label, i, storetype, idx, steal=0):
 				#containers are always clickable
 				Button.SetState (IE_GUI_BUTTON_ENABLED)
 			else:
+				color = {'r': 61, 'g': 47, 'b': 24, 'a': 100}
+				Button.SetBorder (0, color, 1,1)
+				Button.SetBorder (1, color, 1,1)
+				Button.SetBorder (2, color, 1,1)
+				Button.SetBorder (3, color, 1,1)
```

Comment:
Why are you setting all the borders — isn't one enough?

Reply:
I thought that you have to put all the borders to work, I should try with only one. Should it be the number 0 or one specific ?

Reply:
Use the same one we use in the inventory, which would be 2. Just in case the other uses get triggered.

Reply:
Change it to leave only index 2.